### DISCUSSION TITLE
Getting Login Screen in the middle of Upgrade to BigSur using erase-install script

### DISCUSSION THREAD

**Discussion by MehdiYawari** [grahampugh/erase-install#179] [Q&A] [answered]
Hi All
I am using erase-install script on our jamf pro server(triggered by user via Self Service). While upgrading from Catalina to BigSur, after the reboot we always get a really weird Login screen which interrupts the upgrade. (see the Attachment)
![Login-Screen-error](https://user-images.githubusercontent.com/93255458/146936279-51cea9e6-0ba9-429d-ac81-947a7563935a.jpeg)
This is really annoying, because we cant upgrade to BigSur on one try but login first, in order to proceed the installation.
I have also checked, if there any Profile or Policy from our Server, that could produce this problem. So far nothing has been found.
Has anyone experienced this?
Your help is much appreciated.
Thank you
Regards
Mehdi

**Comment by grahampugh** [comment, marked as answer]
That's a feature of some macOS updates, if you have FileVault enabled. Nothing the script can help with.
